Makefile

```
@@ -0,0 +1,36 @@
+.PHONY: start test build mod-tidy lint docker-build docker-run help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)